internal/mode/static/nginx/config/validation/http_njs_match.go

```diff
@@ -46,6 +46,10 @@ func (HTTPNJSMatchValidator) ValidatePathInMatch(path string) error {
 }
 
 func (HTTPNJSMatchValidator) ValidateHeaderNameInMatch(name string) error {
+	if err := k8svalidation.IsHTTPHeaderName(name); err != nil {
+		return fmt.Errorf(err[0])
```

Comment:
since there is no format string used here (ex. `fmt.Errorf("something happened: %v" err[0])`), a better way to is to create an error is errors.New(...)

Reply:
Ah ok, yea I thought using Errorf felt off as I wasn't actually formatting anything and making use of it but didn't know of a better way.